ask-sdk-servlet-support/tst/com/amazon/ask/servlet/SkillServletTest.java

```diff
@@ -65,18 +65,6 @@ public void setup() {
         skill = mock(Skill.class);
     }
 
-    @Test
-    public void doPost_passedVerification_responseSuccess() throws Exception {
```

Comment:
Can you add a test here, where the skill.execute returns null and the asksdkexception gets thrown by the skillservlet class? The current tests mocks the skill.execute to throw an exception which is not true anymore since we have changed the logic in the base dispatcher

Reply:
Added test for this specific case.